drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -229,6 +266,30 @@ void at86rf2xx_set_txpower(const at86rf2xx_t *dev, int16_t txpower)
 #endif
 }
 
+int16_t at86rf2xx_get_rxsensitivity(const at86rf2xx_t *dev)
+{
+    uint8_t rxsens = at86rf2xx_reg_read(dev, AT86RF2XX_REG__RX_SYN)
+                     & AT86RF2XX_RX_SYN__RX_PDT_LEVEL;
+    return rx_sens_to_dbm[rxsens];
+}
+
+void at86rf2xx_set_rxsensitivity(const at86rf2xx_t *dev, int16_t rxsens)
+{
+    rxsens += MIN_RX_SENSITIVITY;
+
+    if (rxsens < 0) {
+        rxsens = 0;
+    }
+    else if (rxsens > MAX_RX_SENSITIVITY) {
+        rxsens = MAX_RX_SENSITIVITY;
+    }
+
+    uint8_t tmp = at86rf2xx_reg_read(dev, AT86RF2XX_REG__RX_SYN);
+    tmp &= ~(AT86RF2XX_RX_SYN__RX_PDT_LEVEL);
+    tmp |= (dbm_to_rx_sens[rxsens] & AT86RF2XX_RX_SYN__RX_PDT_LEVEL);
```

Comment:
@Hyungsin @PeterKietzmann 
First we should realize that `MIN_RX_SENSITIVITY` and `MAX_RX_SENSITIVITY` have negative values (roughly between -100 and -50).

If the incoming value of `rxsense` is greater than `-MIN_RX_SENSITIVITY` (say greater than 220), then it will get the value of `MAX_RX_SENSITIVITY`, which is a negative value.

If the incoming value of `rxsense` is smaller than `-MIN_RX_SENSITIVITY`, then it will get the value zero.

This looks suspicious, no?